### ISSUE TITLE
Swift 2.3 support

### ISSUE THREAD

**Issue by cincas** [sascha/DrawerController#63]
Hi,

Thank you for amazing work!

We are upgrading our project to be compatible with XCode 8.

For now, we decided to use swift 2.3 before we migrate to swift 3. During this process, we found few problems in version `1.1.0`.

```
AnimatedMenuButton.swift:70:69: Value of optional type 'CGPath?' not unwrapped; did you mean to use '!' or '?'?
```

```
AnimatedMenuButton.swift:72:53: Value of optional type 'CGPath?' not unwrapped; did you mean to use '!' or '?'?
```

```
DrawerController.swift:711:91: Value of optional type 'CGPath?' not unwrapped; did you mean to use '!' or '?'?
```

seems like they are just some simple syntax issues, do you have a plan to create a patch for it?
And version `1.1.1` looks like swift 3 version, and we are not quite ready for that.

Do you want us to make a quick PR for this fix and maybe release it as `1.1.0.1`?

Thanks.

**Comment by djxmax** [comment]
+1, I have exactly the same problem.

**Comment by danho322** [comment]
This would be great

**Comment by ganchev-naxex** [comment]
I have the same problem but with swift 3

**Comment by cincas** [comment]
I've made a folk for this fix. 

Just not sure if it is needed.

**Comment by foreldet** [comment]
I've made a fork. It's not a big deal, but it can save some time until we make projects compatible with Swift 3.

**Comment by sascha** [comment]
I've released DrawerController 2.0.0 with Swift 2.3 support and DrawerController 3.0.0 with Swift 3.0 support.
